_api-reference/ingest-apis/ingest-pipelines.md

```diff
@@ -0,0 +1,280 @@
+---
+layout: default
+title: Ingest pipelines
+parent: Ingest APIs
+nav_order: 10
+redirect_from:
+  - /opensearch/rest-api/ingest-apis/create-update-ingest/
+  - /api-reference/ingest-apis/create-update-ingest/
+  - /api-reference/ingest-apis/get-ingest/
+  - /api-reference/ingest-apis/simulate-ingest/
+  - /api-reference/ingest-apis/delete-ingest/
+---
+
+# Ingest pipelines
+
+Ingest pipelines in OpenSearch can only be managed using ingest API operations. When using ingest in production environments, your cluster should contain at least one node with the node roles permission set to `ingest`. For more information about setting up node roles within a cluster, see [Cluster formation]({{site.url}}{{site.baseurl}}/opensearch/cluster/).
+
+To get started, open **Dev Tools**. From the console, you can create, update, test, and delete your pipelines.
+
+## Create or update an ingest pipeline
+
+Creating an ingest pipeline is a vital step in streamlining your data processing workflow. For example, you can enhance data quality, automate data processing tasks, and ensure your data is prepared and optimized for downstream use. Use create pipeline API to create or update pipelines in OpenSearch.
+
+### Path and HTTP method
+```json
+PUT _ingest/pipeline/pipeline-id
+```
+
+The following is an example request to create a pipeline. 
+
+```json
+PUT _ingest/pipeline/pipeline-id
+{
+  "description": "Optional pipeline description",
+  "processors": [
+    {
+      "set": {
+        "description": "Optional processor description",
+        "field": "my-long-field",
+        "value": 10
+      }
+    },
+    {
+      "set": {
+        "description": "Set 'my-boolean-field' to true",
+        "field": "my-boolean-field",
+        "value": true
+      }
+    },
+    {
+      "lowercase": {
+        "field": "my-keyword-field"
+      }
+    }
+  ]
+}
+```
+{% include copy-curl.html %}
+
```

Comment:
@heemin32 Do we want to include content about if a pipeline fails? Are there common reasons why a pipeline fails?